tests/filters/test_addslashes.py

```diff
@@ -31,8 +30,8 @@ def test_addslashes02(self):
     """@setup({"addslashes02": "{{ a|addslashes }} {{ b|addslashes }}"})"""
     template = "{{ a|addslashes }} {{ b|addslashes }}"
 
-    django_template = engines["django"].from_string(template)
-    rust_template = engines["rusty"].from_string(template)
+    django_template = engines["django"].from_string(template)  # noqa
+    rust_template = engines["rusty"].from_string(template)  # noqa
```

Comment:
I missed it when reviewing before, but I think the reason `ruff` is complaining is because `django_template` and `rust_template` are unused.

Both `test_addslashes01` and `test_addslashes02` refer to a `self.engine`, but this doesn't exist because these aren't methods on a unittest test class.

Reply:
See https://github.com/LilyFoote/django-rusty-templates/pull/57